onadata/libs/utils/viewer_tools.py

```diff
@@ -162,20 +162,21 @@ def get_client_ip(request):
     return request.META.get('REMOTE_ADDR')
 
 
-def enketo_url(form_url,
+def get_enketo_urls(form_url,
                id_string,
                instance_xml=None,
                instance_id=None,
                return_url=None,
-               **kwargs):
-    """Return Enketo webform URL."""
+               **kwargs) -> Dict:
```

Comment:
I don't think `Dict` exists in this context.... _Please use Dict from [typing](https://docs.python.org/3/library/typing.html#type-aliases) i.e `Dict[str, str]`_